netcon/onlinedirectipclient/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+set(HEADERS
+  chat_api.h
+  odtclient.h
+  odtstrings.h
+  dip_gametrack.h
+  )
+set(CPPS
+  chat_api.cpp
+  odtclient.cpp
+  ../inetfile/inetgetfile.cpp
```

Comment:
Library already linked with `inetfile` module, is that source file required here?

Reply:
ty, missed that